sage/pallet/src/tests/extrinsics/upgrade_asset_inventory.rs

```diff
@@ -220,14 +220,19 @@ fn upgrade_asset_inventory_should_reject_insufficient_balance() {
 
 #[test]
 fn upgrade_asset_inventory_should_reject_fully_upgraded_storage() {
-	ExtBuilder::default().build().execute_with(|| {
-		PlayerSeasonConfigs::<Test, Instance1>::mutate(ALICE, SEASON_ID_0, |config| {
-			config.inventory_tier = InventoryTier::Max
-		});
+	let alice_initial_balance = MockExistentialDeposit::get() * 10;
```

Comment:
Had to give alice some funds as we first propagate the fee, and then check for the current storage upgrade level.